packages/SwingSet/docs/static-vats.md

```diff
@@ -30,6 +32,8 @@ export function buildRootObject(vatPowers) {
 }
 ```
 
+The root object *must* be an "ephemeral" object, i.e. created with `Far`. It cannot be a virtual or durable object (created with a maker returned by `defineKind` or `defineDurableKind`, or the vat-data convenience wrappers like `prepareSingleton`). This ensures that the root object's identity is stable across upgrade.
```

Comment:
Can this reference liveslots.md and minimize duplication of its contents?
```patch
-`buildRootObject` is expected to return a hardened "Remotable" object with callable methods and no data properties. The best way to do this is with the `Far` function:
+`buildRootObject` must return a hardened "ephemeral" object, as documented in [swingset-liveslots](https://github.com/Agoric/agoric-sdk/blob/master/packages/swingset-liveslots/docs/liveslots.md#buildrootobject).
-
-```js
-import { Far } from '@endo/far';
-export function buildRootObject(vatPowers) {
-  let counter = 0;
-  return Far('root', {
-    increment() {
-      counter += 1;
-    },
-    read() {
-      return counter;
-    }
-  });
-}
-```
-
-The root object *must* be an "ephemeral" object, i.e. created with `Far`. It cannot be a virtual or durable object (created with a maker returned by `defineKind` or `defineDurableKind`, or the vat-data convenience wrappers like `prepareSingleton`). This ensures that the root object's identity is stable across upgrade.
```
(with the ````` ```js ````` block removed from here replacing the one at https://github.com/Agoric/agoric-sdk/blob/master/packages/swingset-liveslots/docs/liveslots.md#buildrootobject since it is a more conventional and illustrative example, or kept here with less prose but still updating swingset-liveslots)

Reply:
yeah, probably a good idea, I think of static-vats.md as slightly more "this is the reference you should be reading while writing your `buildRootObject`, and liveslots.md as "here's how the internals work", but I can't claim I've been consistent with that at all, and this is less likely to bitrot if there are fewer copies of it

Reply:
hm, ok I left in the code example, but removed the extra prose as you said